R/worker.R

```diff
@@ -262,6 +262,12 @@ workhorse = function(iteration, task, learner, resampling, param_values = NULL,
     learner$state$model = NULL
   }
 
+  # repeated saving of the prototype leads to large ResultData objects if the task contains many columns, factor levels or attributes
+  learner$state$data_prototype = NULL
+  learner$state$task_prototype = NULL
+  learner$fallback$state$data_prototype = NULL
```

Comment:
check first whether fallback exists.

Reply:
There is no error even if the fallback does not exist.